train.py

```diff
@@ -143,12 +142,13 @@ def get_input_data_tensors(reader,
     files = gfile.Glob(data_pattern)
     if not files:
       raise IOError("Unable to find training files. data_pattern='" +
-                    data_pattern + "'")
-    logging.info("number of training files: " + str(len(files)))
-    filename_queue = tf.train.string_input_producer(files,
-                                                    num_epochs=num_epochs)
+                    data_pattern + "'.")
+    logging.info("Number of training files: %s.", str(len(files)))
+    filename_queue = tf.train.string_input_producer(
+        files, num_epochs=num_epochs)
```

Comment:
Added the parameter "shuflle=True" to string_input_producer. 

It is the default but it seems that being explicit can improve readability in this case.